### ISSUE TITLE
Test revised GPAD evidence computation

### ISSUE THREAD

**Issue by balhoff** [geneontology/minerva#213]
We would like to explore outputting the evidence for a GPAD annotation as the least common subsumer of the evidence codes used on all the edges in the explanation for an annotation.

Special case: drop the evidence code on the `enabled by` edge before computing the common subsumer.

Possible modification: if an edge has no evidence, we may want to treat it has having some default evidence code for the purpose of the subsumer computation.

**Comment by balhoff** [comment]
@ukemi @vanaukenk we talked about how to combine evidence codes when there are multiple evidences supporting a GP-to-GO annotation. However we didn't go into how to combine the other aspects of evidence: reference, withOrFrom, interactingTaxon, date, etc.

Would we create separate annotations for each different combo, all using our newly computed evidence code? This might require another discussion.

**Comment by vanaukenk** [comment]
@balhoff @ukemi 
Yes, I think we should discuss these additional aspects of the evidence.
Shall we try for some time next week?

**Comment by ukemi** [comment]
I agree. I don't think we took this into consideration at the hackathon. We might want to revisit the controversial 'just use the evidence on the first MF-BP edge for the downstream inferences. Remember the justification for that was that curators already ask for very complicated causal terms without any evidence for the causal relationship in the term.

Just as an aside, I am pouring through the 3000 or so regulates-o-part_of->regulates inferences and I am marking a large number of them invalid. I suspect if we had the time to trace all those annotations down we would find that curators asked for those terms to annotate 'something having a causal effect on a process'. In many cases if we can GO-CAMify the process we would see that the genes annotated to the regulation terms are genes that function in an upstream causal part of a larger process. I will discuss this in an ontology call.

**Comment by vanaukenk** [comment]
Not sure if we have a demo model to work from, but this might be a good candidate:

http://noctua.geneontology.org/editor/graph/gomodel:5941cdbd00000002

**Comment by ukemi** [comment]
Ideally we should have one that branches too.

**Comment by ukemi** [comment]
Never mind, this one does have a branch.

**Comment by vanaukenk** [comment]
I'm going to close this ticket, as this is an older development model and we'd probably make it differently now.
